recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
```

Comment:
Technically don't need a version bump since this doesn't affect packaging, only tests. But not a problem.

Reply:
A new build was needed to resolve #1. (I didn't look deep enough to see why the original build was failing the pip check, but evidently just rebuilding was enough to resolve it)

Thank you for merging!

Reply:
Ah, of course! I should have red #1 more carefully. Makes perfect sense, thanks!